src/embind/embind.js

```diff
@@ -546,6 +547,7 @@ var LibraryEmbind = {
   _embind_register_integer__deps: [
     'embind_repr', '$getShiftFromSize', '$integerReadValueFromPointer',
     '$readLatin1String', '$registerType'],
+  _embind_register_integer__sig: 'vpppjj',
```

Comment:
Looks like the last two arguments are defined as `long`, not `uint64_t`. Wouldn't `j` be incorrect for wasm32 then?

Reply:
Hmm, you are right.   We should int32_t/uint33_t here explicitly since we never want 64-bit values passed to this function (we have _embind_register_bigint for that)